src/app/components/Details.tsx

```diff
@@ -50,3 +50,17 @@ export const Details = styled.details`
     }
   `}
 `;
+
+// Other components than ToC use Details, so we need to style them separately
+// tslint:disable-next-line:variable-name
+export const DetailsTree = styled.a`
```

Comment:
you've left these elements named the way they were before but they no longer really make sense with those names. this component no longer acts like a `details` component because the tree that it expands is not inside of it, its now more of a `CollapseToggle` or something like that.